backend/config/config.go

```diff
@@ -97,6 +97,10 @@ type Backend struct {
 	LitecoinActive bool `json:"litecoinActive"`
 	EthereumActive bool `json:"ethereumActive"`
 
+	// If enabled, the user will see sat/B fees in the fee target priorities, and be able to set a
+	// custom fee in sat/B.
+	ExpertFee bool `json:"expertFee"`
```

Comment:
Shouldn't it go to the `Frontend` part of the config? Unless the idea is to enforce it on the backend and refuse custom fee if this is unset. But maybe I'm missing something.

Reply:
Yeah, I add the custom fee target in the backend based on this, and forward the `expert` field in the same endpoint; see https://github.com/digitalbitbox/bitbox-wallet-app/pull/978.

I also had a version where I did everything in the frontend. Also possible. Not sure which is better, but I think I prefer Backend, so that more business logic is in the backend instead of in the frontend.

Reply:
Ok but one day we'll need to talk about the opaque `Frontend` vs frontend settings in the `Backend` :)

Reply:
sure. but what frontend settings in the backend? Afaik all fields in the Backend config are used by the backend.

Reply:
Like this one in https://github.com/digitalbitbox/bitbox-wallet-app/blob/dfde5c96d162b96cfd6295593a7f03035478388f/frontends/web/src/i18n/config.js#L31-L35

So, the frontend could also be like:

```js
apiGet('config').then(({ frontend }) => { frontend.expertFee }
```

Reply:
Still confused,`userLanguage` is only used in the frontend, `expertFee` is used in the backend. Let's take it offline.

Maybe it makes sense to structure the Frontend anyway instead of putting `interface{}`.